Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -140,6 +140,11 @@ open class NavigationMapView: UIView {
      */
     var pendingCoordinateForRouteLine: CLLocationCoordinate2D?
     
+    /**
+     A custom route line layer position.
+     */
+    var customRouteLineLayerPosition: MapboxMaps.LayerPosition? = nil
```

Comment:
`customRouteLineLayerPosition` is used to remember the custom layer position for route line during active navigation or when route line related properties change.